migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_009__formidling_init.sql

```diff
@@ -0,0 +1,62 @@
+-- Brevbestilling table and sequence
+create sequence if not exists seq_brevbestilling start with 1 increment by 50;
+
+create table brevbestilling
+(
+    id                    bigint primary key,
+    brevbestilling_uuid   uuid                                                                not null unique,
+    saksnummer            character varying(19)                                               not null,
+    dokumentmal_type      character varying(100)                                              not null,
+    template_type         character varying(100),
+    status                character varying(30)                                               not null,
+    dokumentdata          jsonb,
```

Comment:
Jeg forstår det slik at data som legges inn i dokumentdata er for å understøtte behovet til de ulike templatene, og at det tar kort tid fra data lagres til det blir brukt (ikke behov for å migrere data senere). Så et godt case for å bruke ustrukturert :+1:

Reply:
Gir det mening å automatisk slette innholdet her etter at brevet er produsert?

Reply:
enkelte felter for template blir populert av utledede data (eks beregning). Men de feltene som saksbehandler selv legger inn var tanken å putte inn her. F.eks. hvis saksbehandler bestiller et varselsbrev med noe fritekst så vil den gå inn her. Jeg tror ikke det vil bli veldig stort, type large object. Men ja, kan også slette dem hvis det blir behov. 

Jeg har egentlig bare duplisert modellen k9-formidling. Det er mulig vi kan løse det på en annen måte. Kanskje lagre input data i eget tabell.